docs/4_preparing_roms_input_datasets.ipynb

```diff
@@ -146,8 +146,8 @@
      "name": "stdout",
```

Comment:
<p>Looks good - my only comments are to use pathlib for paths consistently with the rest of C-Star, this can be on the next go around though if it's a pain to re-run 👍</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CWorthy-ocean/C-Star/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>